packages/next/server/next-dev-server.js

```diff
@@ -191,6 +198,35 @@ export default class DevServer extends Server {
 
   async run (req, res, parsedUrl) {
     await this.devReady
+    try {
+      await fsStat(join(this.publicDir, '_next'))
```

Comment:
I think this should only be ran when `parsedUrl` includes `/_next`

Reply:
Updated